src/Pages/GlobalConfigurations/DeploymentCharts/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { versionComparatorBySortOrder } from '@Shared/Helpers'
+import { SortingOrder } from '@Common/Constants'
+import { DeploymentChartListDTO, DeploymentChartType, DEVTRON_DEPLOYMENT_CHART_NAMES } from './types'
+import fallbackGuiSchema from './basicViewSchema.json'
+import fallbackJobsNCronJobGuiSchema from './fallbackJobsNCronJobGuiSchema.json'
+
+export const convertDeploymentChartListToChartType = (data: DeploymentChartListDTO): DeploymentChartType[] => {
+    if (!data) {
+        return []
+    }
+    const chartMap: Record<string, DeploymentChartType> = data.reduce(
+        (acc, { id, version, chartDescription: description = '', name, isUserUploaded = true }) => {
```

Comment:
Check if we need default for name, and should we push if id and version is not present?

Reply:
returning early if either of id, version or name are not found